packages/core/admin/admin/src/components/Form.tsx

```diff
@@ -690,15 +690,19 @@ const isErrorMessageDescriptor = (object?: string | object): object is Translati
 /* -------------------------------------------------------------------------------------------------
  * Blocker
  * -----------------------------------------------------------------------------------------------*/
-const Blocker = () => {
+const Blocker = ({ onProceed = () => {} }) => {
```

Comment:
for consistency, can we add an  `onCancel` and write an interface for the types with docs so they appear in the auto-generator and they're helpful?